### ISSUE TITLE
Cannot import CA certificate since december security update

### ISSUE THREAD

**Issue by Catfriend1** [PixelExperience/android-issues#1188]
## Build date

PixelExperience_Plus_dipper-11.0-20211222-1239-OFFICIAL

## Expected Behavior

I go to Android settings, security, certificates, Import certificate from storage, CA certificate, click "yes" on the security info screen, file chooser opens and I choose the crt file, message "CA imported successfully" appears as toast.

## Current Behavior

I go to Android settings, security, certificates, Import certificate from storage, CA certificate, click "yes" on the security info screen, file chooser opens and I choose the crt file, nothing appears OR "Certificate Installer stopped" dialog pops up. The certificate wasn't installed.

Android log says: "could not bind to keychainservice"

## Possible Solution

I don't know.

## Steps to Reproduce

I go to Android settings, security, certificates, Import certificate from storage, CA certificate, click "yes" on the security info screen, file chooser opens and I choose the crt file

/device dipper
/version eleven_plus

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by Dzejzik** [comment]
I'm experiencing same issue
/device dipper
/version eleven_plus
/encrypted

When enabling "Work profile" some company certificates are normally installed. In this case enabling "Work profile" leads to system bootloop in 30sec intervals. Restarting the device and disabling "Work profile" is the only solution to stop bootlooping.

**Comment by GiaSen** [comment]
could you contact me on telegram? I can give you a test build with a possible fix (username is same as github)

**Comment by Catfriend1** [comment]
Hi @GiaSen and thanks for your answer. I tried to revert to a previous twrp full backup and the system did fail the restore. Already had a chat with scherzo on Telegram who told me trwp cannot be used to restore a system with encrypted data partition. I then tried out PE12 official latest release of 2021-12-24 which works well for me on the mi8 and there, the CA import runs fine again. 

I would be willing to help you test out the build, but given the fact that I cannot just backup my fully set-up Android PE12 environment , go to pe11+ for testing and providing you feedback and then restore my 12 setup with twrp, I currently haven't got the time to set up everything again. (I use a lot of apps ...)

So long story short: PE12 is fully working , the bug does not occur there.
